dev/js/pages/UserDashboardView/DocumentView/index.js

```diff
@@ -0,0 +1,152 @@
+  };
+
+  const deleteDocument = (id) => {
+    setUserAddedDocs((prevState) => {
+      return prevState.filter((val) => {
+        return id != val.doc_id;
+      });
+    });
+  };
+
+  const UploadedDocumentNew = (doc) => {
+    return (
+      <UploadedDocument
+        key={doc.doc_id}
+        title={doc.doc_type}
+        date={doc.created_at}
+        content={doc.content}
+        deleteDocument={deleteDocument}
+        id={doc.doc_id}
+      />
+    );
+  };
+
+  const handleSearchDoc = (event) => {
+    setSearchDoc(event.target.value);
+  };
+
+  const openFileUploadModal = () => {
+    setOpenFileUpload(true);
+  };
+
+  const fileuploadclosed = () => {
+    setOpenFileUpload(false);
+  };
+
+  const fileUploadDone = (document) => {
+    const { selectedFile, selectedFileType } = document;
+    const { value } = selectedFileType;
+    uploadUserDocument({ selectedFile, value }, dispatch, (err, data) => {
+      if (data) {
+        setOpenFileUpload(false);
+      }
+      if (err) {
+        console.log(err);
+        setOpenFileUpload(true);
+      }
+    });
+  };
+
+  return (
+    <div className={body}>
+      <div className="mainView">
+        <Header headerText="All Your Documents Are safe with us!">
+          <div className="headerView">
+            <NotificationWidget />
+            <ProfileWidget />
+          </div>
+        </Header>
+        <div className="uploadSection">
+          <form className="uploadedBy">
+            <label className="uploadText">Uploaded By</label>
+            <select
+              className="optionsMenu"
+              onChange={documentOwnerHandler}
+              required
+            >
+              <option value="none" selected disabled hidden>
+                All
+              </option>
+              <option value="you">You</option>
+              <option value="consultant">Consultatnt</option>
+            </select>
+          </form>
+          <form className="searchDocsWrapper">
+            <input
+              className="searchDocs"
+              placeholder="Search Documents"
+              value={searchDoc}
+              onChange={handleSearchDoc}
+            />
+          </form>
+          <button onClick={openFileUploadModal} className="uploadButton">
+            Upload Document
+          </button>
+          {openFileUpload && (
+            <FileUpload
+              fileUploadModalClosed={fileuploadclosed}
+              uploadFile={fileUploadDone}
+            />
+          )}
+        </div>
+        <div className="uploadedDocsWrapper">
+          {userAddedDocs
+            .filter((val) => {
+              if (searchDoc === '') {
+                return val;
+              } else if (
+                val.doc_type.toLowerCase().includes(searchDoc.toLowerCase())
+              ) {
+                return val;
+              }
+            })
+            .map((doc) => {
+              if (documentOwner === '') {
```

Comment:
both if condition return same data either merge it or remove duplicate if condition.

Reply:
Corrected